src/pages/workspaces/workspace/Data.js

```diff
@@ -69,11 +69,23 @@ const styles = {
   }
 }
 
-const DataTypeButton = ({ selected, entityName, children, entityCount, iconName = 'listAlt', iconSize = 14, buttonStyle, after, ...props }) => {
+const SearchResultsPill = ({ count, searching }) => {
```

Comment:
Should the pill have its own tooltip? Currently, hovering over it shows the tooltip for the table, with a row count that may not match the results count.

![Screen Shot 2022-05-17 at 2 17 18 PM](https://user-images.githubusercontent.com/1156625/168883128-052defdd-076f-43b2-85d8-65473cfbb062.png)

Reply:
Hmm, yeah maybe if there is an active cross-table search we can display something like "thing (10 total rows, 1 filtered row)" What do you think?

Reply:
Added in 6473444e567975c124f4fe01998adf9a6b716bae